assets/dev/js/editor/controls/slider.js

```diff
@@ -87,11 +87,9 @@ ControlSliderItemView = ControlBaseUnitsItemView.extend( {
 		const property = this.isMultiple() ? 'sizes' : 'size',
 			value = this.getControlValue( property );
 
-		if ( value && '' !== value ) {
-			return value;
-		}
-
-		return this.model.get( 'placeholder' )?.[ property ];
+		return value ||
```

Comment:
Is it related to the Widescreen breakpoint?

Reply:
> Is it related to the Widescreen breakpoint?

Reverted.